portindex2json/portindex2json.tcl

```diff
@@ -90,5 +105,11 @@ while {[gets $fd line] >= 0} {
     lappend objects [::json::write object {*}[array get json_portinfo]]
 }
 
+if {[array exists json_info]} {
+    set json_output(info) [::json::write object {*}[array get json_info]]
+}
+
+set json_output(ports) [::json::write array {*}$objects]
+
 chan configure stdout -encoding utf-8
-puts [::json::write array {*}$objects]
+puts [::json::write object {*}[array get json_output]]
```

Comment:
Just nitpicking about the nice habit of keeping one empty line at the end of the file.

Reply:
😞 And I missed it again.